src/pudl/output/pudltabl.py

```diff
@@ -116,15 +116,14 @@ def __init__(
 
         # grab all working eia dates to use to set start and end dates if they
         # are not set
-        eia_dates = pudl.helpers.get_working_eia_dates()
         if start_date is None:
-            self.start_date = min(eia_dates)
+            self.start_date = min(pudl.helpers.get_working_dates_by_datasource("ferc1"))
```

Comment:
Yes, exactly. I mentioned this in the big pr comment, but I think the right way to do this would be to have this check what the minimum possible start date is from every data source instead of hard-coding it to be FERC.

Reply:
Sounds good. Maybe good to leave a comment or TODO here, but definitely not blocking!